mk.scm

```diff
@@ -473,6 +473,11 @@
   (lambda (ground-atom term)
     (lambdag@ (st)
       (let ((term (walk term (state-S st))))
+        (unless (or (symbol? ground-atom)
+                    (number? ground-atom)
+                    (boolean? ground-atom)
+                    (null? ground-atom))
+          (error 'absento "first argument to absento must be a ground atom"))
```

Comment:
Is there a reason this check is inside here instead of just inside the outer lambda?